src/mcom.md

```diff
@@ -1,6 +1,6 @@
 ---
 ee_only: true
-title: MCOM
+title: Magento Commerce Order Management (MCOM)
```

Comment:
This looks good! Did not review the whole PR.

Reply:
Thanks @shrielenee !, you were not supposed to, thanks!